network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/space_card_listing_block.html

```diff
@@ -0,0 +1,27 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div class="container tw-py-4 medium:tw-py-7">
+  <h3 class="tw-font-semibold tw-mb-4">{{ value.title }}</h3>
+  <div class="tw-grid tw-grid-cols-1 large:tw-grid-cols-2 tw-gap-6 large:tw-gap-10">
+    {% for space in self.space_cards %}
+      <a class="tw-group tw-no-underline tw-p-5 tw-border tw-border-gray-60/20 tw-text-[22px] tw-flex medium:tw-flex-row tw-flex-col medium:tw-space-x-4"
+         href="{% if space_card.link_page %}{{ space_card.link_page.url }}{% elif space_card.link_url %}{{ space_card.link_url }}{% endif %}">
```

Comment:
Is there a scenario where there won't be a link?

Reply:
This should always be a link :)